src/component/product/Productlisting.jsx

```diff
@@ -15,8 +15,11 @@ const Productlisting = ({ product }) => {
     addToCartHandler,
     cartState: { cartData },
   } = useCart();
-  const { setWishCount, setWishList, wishList } = useWishlist();
-  const isInWishList = (id) => wishList.find(({ _id }) => _id == id);
+  const {
+    wishlistState: { wishlistData },
+    addToWishlistHandler,
+  } = useWishlist();
+  const isInWishList = (id) => wishlistData.find(({ _id }) => _id == id);
   const isInCartList = (id) => cartData.find(({ _id }) => _id == id);
   var classNames = require("classnames");
```

Comment:
I got confused here. Can you tell what you are trying to do with this require('classname"). If you are importing it from package then its not the right way

Reply:
I used classnames package here basically to concatenate classes based on given condition. You can read more about it here [classnames](https://www.npmjs.com/package/classnames)